<?xml version="1.0" encoding="UTF-8" standalone="no"?><bugrepository name="FLINK">
  <bug fixdate="2023-6-31 01:00:00" id="32231" opendate="2023-5-31 00:00:00" resolution="Fixed">
    <buginformation>
      <summary>libssl not found when running CI</summary>
      <description>--2023-05-31 19:10:13-- http://security.ubuntu.com/ubuntu/pool/main/o/openssl1.0/libssl1.0.0_1.0.2n-1ubuntu5.12_amd64.debResolving security.ubuntu.com (security.ubuntu.com)... 185.125.190.39, 91.189.91.38, 91.189.91.39, ...Connecting to security.ubuntu.com (security.ubuntu.com)|185.125.190.39|:80... connected.HTTP request sent, awaiting response... 404 Not Found2023-05-31 19:10:13 ERROR 404: Not Found.e.g.https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=49523&amp;view=logs&amp;j=bea52777-eaf8-5663-8482-18fbc3630e81&amp;t=d6e79740-7cf7-5407-2e69-ca34c9be0efb&amp;l=265</description>
      <version>1.18.0,1.16.3,1.17.2</version>
      <fixedVersion>1.18.0,1.16.3,1.17.2</fixedVersion>
      <type>Technical Debt</type>
    </buginformation>
    <fixedFiles>
      <file type="M">tools.azure-pipelines.e2e-template.yml</file>
    </fixedFiles>
  </bug>
  <bug fixdate="2023-6-1 01:00:00" id="32232" opendate="2023-6-1 00:00:00" resolution="Fixed">
    <buginformation>
      <summary>Supports parse truncate table statement</summary>
      <description/>
      <version>None</version>
      <fixedVersion>None</fixedVersion>
      <type>Sub-task</type>
    </buginformation>
    <fixedFiles>
      <file type="M">flink-table.flink-sql-parser.src.test.java.org.apache.flink.sql.parser.FlinkSqlParserImplTest.java</file>
      <file type="M">flink-table.flink-sql-parser.src.main.codegen.includes.parserImpls.ftl</file>
      <file type="M">flink-table.flink-sql-parser.src.main.codegen.data.Parser.tdd</file>
    </fixedFiles>
  </bug>
  
  <bug fixdate="2023-6-20 01:00:00" id="32392" opendate="2023-6-20 00:00:00" resolution="Fixed">
    <buginformation>
      <summary>Several jobs failed on AZP with No space left on device</summary>
      <description>This Build failed with no space left https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=50162&amp;view=logs&amp;j=585d8b77-fa33-51bc-8163-03e54ba9ce5b##[error]Unhandled exception. System.IO.IOException: No space left on device : '/home/vsts/agents/3.220.5/_diag/Worker_20230619-021757-utc.log' at System.IO.RandomAccess.WriteAtOffset(SafeFileHandle handle, ReadOnlySpan`1 buffer, Int64 fileOffset) at System.IO.Strategies.BufferedFileStreamStrategy.FlushWrite() at System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder) at System.Diagnostics.TextWriterTraceListener.Flush() at Microsoft.VisualStudio.Services.Agent.HostTraceListener.WriteHeader(String source, TraceEventType eventType, Int32 id) in /home/vsts/work/1/s/src/Microsoft.VisualStudio.Services.Agent/HostTraceListener.cs:line 151 at Microsoft.VisualStudio.Services.Agent.HostTraceListener.TraceEvent(TraceEventCache eventCache, String source, TraceEventType eventType, Int32 id, String message) in /home/vsts/work/1/s/src/Microsoft.VisualStudio.Services.Agent/HostTraceListener.cs:line 81 at System.Diagnostics.TraceSource.TraceEvent(TraceEventType eventType, Int32 id, String message) at Microsoft.VisualStudio.Services.Agent.Util.ProcessInvoker.ProcessExitedHandler(Object sender, EventArgs e) in /home/vsts/work/1/s/src/Agent.Sdk/ProcessInvoker.cs:line 496 at System.Diagnostics.Process.OnExited() at System.Diagnostics.Process.RaiseOnExited() at System.Diagnostics.Process.CompletionCallback(Object waitHandleContext, Boolean wasSignaled) at System.Threading._ThreadPoolWaitOrTimerCallback.WaitOrTimerCallback_Context_f(Object state) at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)--- End of stack trace from previous location --- at System.Threading._ThreadPoolWaitOrTimerCallback.PerformWaitOrTimerCallback(_ThreadPoolWaitOrTimerCallback helper, Boolean timedOut) at System.Threading.PortableThreadPool.CompleteWait(RegisteredWaitHandle handle, Boolean timedOut) at System.Threading.ThreadPoolWorkQueue.Dispatch() at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart(),##[error]The hosted runner encountered an error while running your job. (Error Type: Failure).for 1.16, 1.17 it happens while 'Upload artifacts to S3'for 1.18 while 'Deploy maven snapshot'</description>
      <version>1.18.0,1.16.3,1.17.2</version>
      <fixedVersion>1.18.0,1.16.3,1.17.2</fixedVersion>
      <type>Bug</type>
    </buginformation>
    <fixedFiles>
      <file type="M">tools.azure-pipelines.jobs-template.yml</file>
      <file type="M">tools.azure-pipelines.e2e-template.yml</file>
      <file type="M">tools.azure-pipelines.build-nightly-dist.yml</file>
      <file type="M">tools.azure-pipelines.build-apache-repo.yml</file>
      <file type="M">azure-pipelines.yml</file>
    </fixedFiles>
  </bug>
  
  <bug fixdate="2023-7-12 01:00:00" id="32585" opendate="2023-7-12 00:00:00" resolution="Fixed">
    <buginformation>
      <summary>Filter javax.xml.bind:jaxb-api false positive for Pulsar connector</summary>
      <description/>
      <version>None</version>
      <fixedVersion>None</fixedVersion>
      <type>Improvement</type>
    </buginformation>
    <fixedFiles>
      <file type="M">tools.ci.flink-ci-tools.src.main.java.org.apache.flink.tools.ci.licensecheck.JarFileChecker.java</file>
    </fixedFiles>
  </bug>
  
</bugrepository>